packages/client/public/layout/layout.js

```diff
@@ -0,0 +1,39 @@
+const useBrowser = chrome || browser;
```

Comment:
`useBrowser` implies a boolean to me which this is not. Would something like `agent` or `activeBrowser` or something work?